script/Arcueid/Day 7/07_00_ARC07_3D.txt

```diff
@@ -65,7 +65,8 @@ Arcueid looks straight at me, pinning me down with her gaze.
 [sha:4803809f4da556d4a9a30cc8da160ab4d67ed59b]{
 -- Page 232, Offset 17136.
 -- こっちが少しでも動けば、即座に襲いかかってくる、そんな視線だ。
-It was the sort of stare that made me feel she'd swoop down on me the instant I move a muscle.
+//Not sure what the tense was in JP, switched entirely to past
```

Comment:
japanese was present tense, but tense promotion isn't a bad thing if it makes the english clearer.